src/Fixer/Basic/BracesFixer.php

```diff
@@ -429,7 +439,11 @@ function ($item) {
                     || !($nextToken->isWhitespace() && $nextToken->isWhitespace(" \t"))
```

Comment:
while you're at it ;) can you please wrap the `($nextToken [...] && [...] "\n") // preserve blank lines` 
within `(` `)`

and update `// set indent only if it is not a case, when comment is following { in same line` to `on the same line`